src/GuiHandler.java

```diff
@@ -0,0 +1,373 @@
+        btnTryALetter.setPrefWidth(150);
+        btnTryALetter.setLayoutY(180);
+        btnTryALetter.setLayoutX(80);
+        btnTryALetter.setOnAction(event -> checkLetter() );
+
+        lblTryAWord = new Label();
+        lblTryAWord.setText("Try a word:");
+        lblTryAWord.setLayoutY(220);
+        lblTryAWord.setLayoutX(10);
+
+        txtTryAWord = new TextField();
+        txtTryAWord.setPrefWidth(150);
+        txtTryAWord.setLayoutY(220);
+        txtTryAWord.setLayoutX(80);
+
+        btnTryAWord = new Button();
+        btnTryAWord.setText("LET'S TRY THIS WORD");
+        btnTryAWord.setPrefWidth(150);
+        btnTryAWord.setLayoutY(250);
+        btnTryAWord.setLayoutX(80);
+        btnTryAWord.setOnAction(event -> checkWord());
+        
+        lblLettersCorrect = new Label();
+        lblLettersCorrect.setText("Letters correct: ");
+        lblLettersCorrect.setLayoutY(350);
+        lblLettersCorrect.setLayoutX(10);
+
+        lblLettersWrong = new Label();
+        lblLettersWrong.setText("Letters wrong: ");
+        lblLettersWrong.setLayoutY(380);
+        lblLettersWrong.setLayoutX(10);
+
+        imgHangman = new ImageView();
+        imgHangman.setImage( new Image( getClass().getResource("img/hangman_0.jpg").toString() ) );
+        imgHangman.setFitWidth(110);
+        imgHangman.setPreserveRatio(true);
+        imgHangman.setLayoutY(150);
+        imgHangman.setLayoutX(350);
+        
+        gameEndText = new Label("");
+        gameEndText.setLayoutY(285);
+        gameEndText.setLayoutX(325);
+        gameEndText.setVisible(false);
+        
+        guesWordLenght = new Label("");
+        guesWordLenght.setLayoutY(285);
+        guesWordLenght.setLayoutX(325);
+        guesWordLenght.setVisible(false);
+        
+        reset = new Button("reset");
+        reset.setLayoutY(325);
+        reset.setLayoutX(325);
+        reset.setVisible(false);
+        reset.setOnAction(event -> reset());
+
+        correctLetters = "";
+        
+        //Voeg de controls toe aan de pane
+        pane.getChildren().addAll(lblSetAnswer, txtSetAnswer, btnSetAnswer, shapeLine);
+        pane.getChildren().addAll(lblRules, lblTryALetter, txtTryALetter, btnTryALetter, lblTryAWord, txtTryAWord, btnTryAWord, lblLettersCorrect, lblLettersWrong, imgHangman, gameEndText, reset, guesWordLenght);
+    }
+
+    public Pane getPane()
+    {
+        return pane;
+    }
+
+    /**
+     * Alles voor de logica (Controller)
+     */
+    public void setAnswer()
+    {
+    	if(txtSetAnswer.getText().matches(".*\\d.*")) {errorPopupHandler(0, "", "Het woord mag alleen letters bevatten"); return;}
+    	
+    	int answerLenght = txtSetAnswer.getText().length();
+    	if(answerLenght < 3) 
+    	{
+    		errorPopupHandler(0, "", "Het woord moet minimaal 3 letters hebben");
+    		return;
+    	}
+    	else if(answerLenght > 8)
+    	{
+    		errorPopupHandler(8, "Het woord mag maximaal 8 letters lang zijn", "");
+    		return;
+    	}
```

Comment:
```suggestion
        int answerLenght = txtSetAnswer.getText().length();
        string error = "";

        //error handling by doing it with a string we make sure the player sees all his mistakes so he doesn't need to do it over and over and over again till he fought all errors.
        if(txtSetAnswer.getText().matches(".*\\d.*")) error += "Het woord mag alleen letters bevatten \n\n";
        if(answerLenght < 3) error += "Het woord moet minimaal 3 letters bevatten \n\n";
        else if(answerLenght > 8) error += "Het woord mag maximaal maar 8 leters bevatten \n\n";

        if(error.isNotEmpty()){
            errorPopupHandler(0, "", error);
            return;
        }
```

Wat als we een error string maken en vervolgens kijken of die string errors heeft gekregen door te kijken of die leeg is
dit maakt het kleiner en je kan meteen alle errors die komen zien ipv elke error per keer.